### ISSUE TITLE
invalid GeV unit 

### ISSUE THREAD

**Issue by jkotan** [SciCatProject/frontend#979]
### Summary

GeV unit is not allowed. You cannot change quantity value with the web server in the Edit widget.

### Steps to Reproduce

Try to change Value of quantity with  the GeV unit.

### Current Behaviour

After accepting changes `Invalid unit` is displayed

### Expected Behaviour

One should be able to change a quantity value with the GeV unit.

### Extra Details

I attach screenshot
![gev-invalid-unit](https://user-images.githubusercontent.com/10651163/184139405-adde31cd-184b-49f0-9720-71fcf5e41cad.png)
.

**Comment by nguyenlinhlinh** [comment]
There is a list of predefined units which Scicat user to check if the unit is valid or not. For eV I can see eV and keV. I think we can add other unit prefix like M, G, T, P, E, Z Y too. What do you think @nitrosx and @stephan271 ?

**Comment by stephan271** [comment]
I was expecting that all prefixes are supported for all units. Isnt that the case ?

**Comment by nguyenlinhlinh** [comment]
No, the list has units with some common prefixes but not all prefixes. See [here](https://github.com/SciCatProject/frontend/blob/8cc77aa1b333e6149393e40e3e1b2ae98592bf62/src/app/shared/services/units.service.ts#L7)

**Comment by jkotan** [comment]
For me it is weird eV and keV are treated as length units instead of energy units.
